tests/test_cadquery.py

```diff
@@ -2957,7 +2957,7 @@ def test_assembleEdges(self):
         edge_wire = edge_wire.add(
             Workplane("YZ")
             .workplane()
-            .transformed(offset=Vector(0, 0, -7), rotate=Vector(0, 45, 0))
+            .transformed(offset=Vector(0, 0, -7), rotate=Vector(45, 0, 0))
```

Comment:
I'm not sure why the 45 and 0 here switched are swapped. Was that intentional?

Reply:
It made the test pass. I think it is related to the changes making rotation relative to the local coordinate system. Here you have a visual comparison of the two
![obraz](https://user-images.githubusercontent.com/13981538/82155635-9a450980-9876-11ea-9178-343d7b6cea55.png)